data/json/items/gun/exodii.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "id": "pamd68",
+    "looks_like": "ar15",
```

Comment:
Not having a copy-from: [abstract] means there's no modslots on this.
![image](https://user-images.githubusercontent.com/51099123/85749090-8c38a300-b6d6-11ea-892e-c876157d5545.png)

Reply:
It's not supposed to have mod slots.

Reply:
Not even mod _mount_ slots?

Reply:
It's debatable if it should even have a human wieldable version